ci/scripts/execute_build_examples.sh

```diff
@@ -17,7 +17,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -e
+set -ex
```

Comment:
just to confirm, was this change intended, or "debugging" that accidentally got left in?

Reply:
It was, and leaks no secrets.